ENIGMAsystem/SHELL/Widget_Systems/TinyFileDialogs/Makefile

```diff
@@ -1,2 +1,9 @@
 include $(SHARED_SRC_DIR)/tinyfiledialogs/Makefile
-SOURCES += $(wildcard Widget_Systems/TinyFileDialogs/*.cpp)
+
+ifeq ($(TARGET-PLATFORM), MacOSX)
+	SOURCES += $(wildcard Widget_Systems/TinyFileDialogs/dialogs-macos.cpp)
```

Comment:
wildcard is for when you have * you can just use SOURCES += Widget_Systems/TinyFileDialogs/dialogs-macos.cpp here